repository-data/webfiles/src/main/resources/site/scss/intranet/components/_intra-home-article-grid.scss

```diff
@@ -27,6 +27,13 @@
         &:nth-child(3n+3) {
             padding-left: 10px;
         }
+
+        /*@include media($max-width: $min-desktop-width) {
+            max-height: 350px;
+            width: 50%;
+            padding-left: 10px;
+            padding-right: 10px;
+        }*/
```

Comment:
Is this needed soon? Otherwise, it's probably better not to commit it.

Reply:
I have removed for now. Saved for later.